lib/components/progress-bar/index.tsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
-import { LinearProgress } from '@material-ui/core';
+import { LinearProgress, LinearProgressProps } from '@material-ui/core';
 
-const ProgressBar = props => {
+const ProgressBar: React.FunctionComponent<LinearProgressProps> = props => {
```

Comment:
Did you try this?

```ts
const ProcessBar: LinearProgress = props => …
```

Reply:
<img width="1100" alt="Screen Shot 2020-04-17 at 4 51 10 PM" src="https://user-images.githubusercontent.com/6817400/79612977-aac56100-80cb-11ea-9b30-973481a969db.png">

Reply:
this works for me

```ts
const ProgressBar: typeof LinearProgress = props => {
	…
}
```